### PR TITLE
1500 remove Faq code

### LINKED ISSUE #1500: Remove code from client project related to FAQs
### Overview
There are several React components in the client code related to a partially-implemented FAQ feature. We do not have immediate plans to complete this implementation and should delete the corresponding router link(s) and react components.

### Action Items
- [ ] Delete obsolete FAQ code
- [ ] Do not delete Web  API endpoints, database tables, etc. in  case we choose to re-visit this implementation later.
- [ ] Uninstall any NPM packages that were only used in support of FAQs (maybe react-quill, etc.)

### PR DESCRIPTION
**Pull request by devnikesh**
Fixes #1500

### CODE REVIEW

**Comment by hanapotski** [comment]
I think we want to keep `StaticPagesHI` 🤔

File: client/src/App.js [hunk 1]
```diff
@@ -281,9 +280,6 @@ function App() {
                           <Route path="/about">
                             <About />
                           </Route>
-                          <Route exact path="/faqs">
-                            <Faq />
-                          </Route>
```
Comment (hanapotski) [inline client/src/App.js:286]: We also want to keep this

File: client/src/components/Faq.js [hunk 2]
```diff
@@ -1,14 +0,0 @@
-import React from "react";
```
Comment (hanapotski) [inline client/src/components/Faq.js:1]: we want to keep this file as well

File: client/src/helpers/Constants.js [hunk 3]
```diff
@@ -2,7 +2,6 @@ export const MENU_ITEMS = [
   { text: "Find Food", link: "/organizations" },
   { text: "About Us", link: "/about" },
   { text: "Donate", link: "/donate" },
-  { text: "FAQs", link: "/faqs" },
```
Comment (hanapotski) [inline client/src/helpers/Constants.js:5]: we want to keep this too

Comment (devnikesh) [reply client/src/helpers/Constants.js]: Thanks Hannah, I reverted those changes and did it as suggessted.  😄

**Review by hanapotski — APPROVED** [review]

**Pull request merged**